cpp/experiments/employee_factory/source/employee.h

```diff
@@ -0,0 +1,12 @@
+#ifndef EMPLOYEE_FACTORY_EMPLOYEE_H
+#define EMPLOYEE_FACTORY_EMPLOYEE_H
+
+#include <string>
+
```

Comment:
Also maybe consider using a namespace for your code. Perhaps putting your code in a directory with a name that matches the namespace and having your main.cpp be outside of the dir/namespace?

To me it looks much cleaner, easier to find the entrypoint.

Reply:
Done